### PR TITLE
New photo carousel

### PR DESCRIPTION
**Pull request by Jvenci**
Hello @aku2nnya, this commit is my MVP 

Implements the photo carousel functionality.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
         "browser": true,
-        "es6": true
+        "es6": true,
+        "jest": true
     },
-    "extends": ["airbnb"],
+    "extends": [
+        "airbnb"
+    ],
     "globals": {
```
Comment (aku2nnya) [inline .eslintrc.json]: Not sure about sourceType, but believe Atomics and SharedArrayBuffer isn't necessary?

File: __mocks__/fileMock.js [hunk 2]
```diff
@@ -0,0 +1 @@
+module.exports = 'test-file-stub';
```
Comment (aku2nnya) [inline __mocks__/fileMock.js]: Please add new linebreak to meet style guide

File: __mocks__/styleMock.js [hunk 3]
```diff
@@ -0,0 +1 @@
+module.exports = {};
```
Comment (aku2nnya) [inline __mocks__/styleMock.js]: Please add new linebreak to meet style guide

File: client/src/App.jsx [hunk 4]
```diff
@@ -6,25 +6,48 @@ class App extends Component {
...
+        next: res.data.slice(3),
         previous: [],
       }))
       .catch(err => console.error('Unable to complete: ', err));
   }
+
   handleNext() {
-    this.state.previous.length === 9 ? 'conditional render arrow away'
+    const { previous, next, currentDisplay } = this.state;
+    const newPrev = [...previous, currentDisplay.shift()];
```
Comment (aku2nnya) [inline client/src/App.jsx]: nice

File: client/src/Homes.jsx [hunk 5]
```diff
@@ -2,37 +2,71 @@ import React from 'react';
 import './style.css';
 import Stars from './Stars';
 
-const Homes = (props) => {
-  const { data } = props;
-  const listing = data.map((home) => {
+const Homes = ({ handleNext, data, prev, next, handlePrev }) => {
+  const listing = data.map(home => (
```
Comment (aku2nnya) [inline client/src/Homes.jsx]: Getting a unique key warning from react on console?

File: client/src/style.css [hunk 6]
```diff
@@ -4,8 +4,6 @@
   height: 338px;
   flex-direction: row;
```
Comment (aku2nnya) [inline client/src/style.css]: I believe flex-direction at "row" is the default behavior, so this is unnecessary?

File: .eslintrc.json (hunk 1 above)
Comment (Jvenci) [reply .eslintrc.json]: That was auto-generated by eslint

File: client/src/App.jsx (hunk 4 above)
Comment (Jvenci) [reply client/src/App.jsx]: thank you!

**Pull request merged**